src/resolve/resolved-dnstls-openssl.c

```diff
@@ -392,8 +392,8 @@ int dnstls_manager_init(Manager *manager) {
 
         assert(manager);
 
-        ERR_load_crypto_strings();
-        SSL_load_error_strings();
+        OPENSSL_init_crypto(OPENSSL_INIT_LOAD_CRYPTO_STRINGS, NULL);
+        OPENSSL_init_ssl(OPENSSL_INIT_LOAD_SSL_STRINGS | OPENSSL_INIT_LOAD_CRYPTO_STRINGS, NULL);
```

Comment:
Both functions return an error (1 on success or 0 on error), I think this should be checked.

Reply:
And to answer the question whether we want this at all: sounds like we don't need to call this. So maybe just drop the whole thing altogether and see if tests pass?